resources/queries/cds/ds_visittag.sql

```diff
@@ -18,13 +18,9 @@ DISTINCT arm_visit.study_arm_visit_type AS Name,
 arm_visit.study_arm_visit_type AS caption,
 FALSE AS SingleUse
 FROM cds.import_studypartgrouparmvisit AS arm_visit
-WHERE study_arm_visit_type IS NOT NULL
+WHERE study_arm_visit_type IS NOT NULL AND study_arm_visit_type != 'Placeholder'
 
-UNION
-
-SELECT
-DISTINCT arm_visit.study_arm_visit_label AS Name,
-arm_visit.study_arm_visit_label AS caption,
-TRUE AS SingleUse
-FROM cds.import_studypartgrouparmvisit AS arm_visit
-WHERE study_arm_visit_label IS NOT NULL
\ No newline at end of file
+-- add the pre-defined alignment visits tags
+UNION ALL SELECT 'Enrollment' AS Name, 'Enrollment' AS caption, TRUE AS SingleUse
```

Comment:
NVM, with tags from study_arm_visit_type also has False as SingleUse, this sql will never result in duplicate primary key error. Union All and Union will function just the same in this case.